scripts/smoke-test-demo

```diff
@@ -29,17 +29,31 @@ function wait_for() {
 	echo "Endpoint $what @ $url is Ok, after $elapsed seconds"
 }
 
+# usage: get_balance <address> <block>
+function get_balance() {
+	(
+		unset MNEMONIC
+		cargo run --release --all-features --bin bridge -- balance -e $SEQUENCER_API -a $1 -b $2
```

Comment:
Not a huge deal but I think it's kind of nice that this script is currently portable and doesn't need a rust toolchain. Maybe it would be useful to expose reading the bridge APIs via the explorer then we could query that instead. If we do add the `cargo run` here we should probably add rust-cache to the `test-demo` github action workflow.

Reply:
What do you mean exactly by reading the bridge APIs via the explorer? 

Another approach could be to use `bridge` as an executable for the native demo (since the native demo already relies on cargo targets being added to `$PATH`) and use the `bridge` docker for the docker demo, which would also have the nice benefit of testing that the new docker is built correctly. I could add a CLI flag to this script to choose the bridge executable

Reply:
I mean if we had a HTTP API that allowed to query the bridge balances either run by the hotshot node or via the block explorer that would be useful here and I think it would be convenient with anyone interacting with our sequencer too but it's also somewhat out of scope.

We could also call `bridge` if it's in the PATH and otherwise try the docker container then we don't need a flag.

Reply:
> I mean if we had a HTTP API that allowed to query the bridge balances either run by the hotshot node or via the block explorer that would be useful here and I think it would be convenient with anyone interacting with our sequencer too but it's also somewhat out of scope.

I agree, but yeah, I think adding this to the block explorer is a lot of work. We do have an HTTP API (that is how the `bridge` tool gets its information) but it outputs a merkle path and the balance is base64 encoded which makes it a bit annoying to work with, hence the `bridge` tool.

Reply:
[61c3086](https://github.com/EspressoSystems/espresso-sequencer/pull/1390/commits/61c3086869a5f30324df84cc1b6b9b646fb57041)